python/ray/_private/test_utils.py

```diff
@@ -2103,3 +2103,12 @@ def _execute_command_on_node(command: str, node_ip: str):
     except subprocess.CalledProcessError as e:
         print("Exit code:", e.returncode)
         print("Stderr:", e.stderr)
+
+
+RPC_FAILURE_MAP = {
+    "request": "100:0:0",
+    "response": "0:100:0",
+    "in_flight": "0:0:100",
```

Comment:
seems our codebase uses it as one word?

Reply:
definitely two words: "the requests are in flight" and "the in-flight requests"